### DISCUSSION TITLE
p-tree gets extra numbers (number of sub-trees)

### DISCUSSION THREAD

**Discussion by tim-wilkins** [primefaces/community#530] [PrimeNG] [answered]
Visible on https://primeng.org/tree I get numbers after a section title (e.g. "Documents 2" for first sample).  Is this left over from testing?  In any case please can I have a way to remove them?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue it would probably left over from testing. Removed right now and the live will be fixed by tomorrow's release
